client/extensions/woocommerce/app/settings/payments/payment-method-item.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import ListItem from '../../../components/list/list-item';
+import ListItemField from '../../../components/list/list-item-field';
+
+const PaymentMethodItem = ( { method, translate } ) => {
+	return (
+		<ListItem>
+			<ListItemField>
+				{
+					method.isSuggested &&
+					(
+						<p className="payments__method-suggested">
+							{ translate( 'Suggested Method' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **73** times:
`translate( 'Suggested media' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).